cmd/minikube/cmd/start_test.go

```diff
@@ -43,3 +45,22 @@ func Test_extractVMDriverVersion(t *testing.T) {
 		t.Errorf("Expected version: %s, got: %s", expectedVersion, v)
 	}
 }
+
+func Test_validateOSSupportVMDriver(t *testing.T) {
+	tests := []struct {
+		driver   string
+		OS       string
+		supports bool
+	}{
+		{constants.DriverNone, constants.Darwin, false},
+		{constants.DriverNone, constants.Windows, false},
+		{constants.DriverNone, constants.Linux, true},
```

Comment:
Could add some tests for e.g. VirtualBox here ?

Reply:
of course! 

virtualbox is supported on all envs?
hyperkit darwin only?
kvm2 anything but darwin or widnows?

Reply:
I think vbox is ok/enough, it's not like we have any logic for the others. basically "not none"

it is supported on all three platforms, yes (if installed/without hyper-v, etc.etc but anyway)

Reply:
@afbjorklund I can add logic to those if it makes sense.

Reply:
I suppose we could lock the other three to each platform, basically kvm2 = linux